actions/conventional_commit/main.go

```diff
@@ -85,6 +95,37 @@ func main() {
 	exitCode = 0
 }
 
+// allowBypass checks if the commit author is allowed to bypass the conventional commit
+// check.
+func allowBypass(commit *github.RepositoryCommit) bool {
+	// Read emails from BYPASS_AUTHOR_EMAILS env var in bypassAuthorEmails.
+	//
+	// Note: This is here so the unit tests can set this.
```

Comment:
Why isn't it just a function parameter if it's just for unit tests? Or is this how the argument is passed-in by github?

Reply:
We pass it in as an env var through the action, but we also do the same in the unit tests to test that functionality. This comment means it's in this function instead of `RunAction` because unit tests only see this function.

I could move it out to still be a param, but wanted to test the env parsing functionality here as well.

Reply:
I guess I misunderstood the scope of the comment!
```suggestion
	// Note: This parsing is here to make it easy to unit test.
```

Reply:
Oops, merged too early, I'll go add this 👀